<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManualCAD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/Resource/M.png</normaloff>:/icons/Resource/M.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>绘图引擎</string>
    </property>
    <addaction name="actionUseStupid"/>
    <addaction name="actionUseQtGui"/>
    <addaction name="actionUseOpenGL"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>放置</string>
    </property>
    <addaction name="actiona_2"/>
    <addaction name="actionb_2"/>
    <addaction name="actionc"/>
    <addaction name="actions"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiona"/>
    <addaction name="actionb"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actions_2"/>
    <addaction name="actiond"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu1"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiona_2"/>
   <addaction name="actionb_2"/>
   <addaction name="actionc"/>
   <addaction name="actions"/>
   <addaction name="separator"/>
   <addaction name="actions_2"/>
   <addaction name="actiond"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUseStupid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stupid模式</string>
   </property>
  </action>
  <action name="actionUseQtGui">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Qt内嵌模式</string>
   </property>
  </action>
  <action name="actionUseOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGL模式</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actiona_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/line.png</normaloff>:/icons/Resource/line.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="actionb_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/rectangle.png</normaloff>:/icons/Resource/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="actionc">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/circle.png</normaloff>:/icons/Resource/circle.png</iconset>
   </property>
   <property name="text">
    <string>圆形</string>
   </property>
  </action>
  <action name="actions">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/text.png</normaloff>:/icons/Resource/text.png</iconset>
   </property>
   <property name="text">
    <string>文字</string>
   </property>
  </action>
  <action name="actions_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/select.png</normaloff>:/icons/Resource/select.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="actiond">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/delete.png</normaloff>:/icons/Resource/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
